packages/go_router/lib/src/matching.dart

```diff
@@ -30,7 +30,11 @@ class RouteMatcher {
       routes: configuration.routes,
       parentFullpath: '',
       parentSubloc: '',
+      // Here, if the request is `a/b/?q1=v1&q2=v2&q2=v3`;
```

Comment:
This comment feel a bit out of place, maybe put this is doc string of the class property instead?